config/plugins/visualizations/openseadragon/config/openseadragon.xml

```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE visualization SYSTEM "../../visualization.dtd">
+<visualization name="OpenSeadragon">
+    <description>Viewer for high-resolution zoomable images.</description>
+    <data_sources>
+        <data_source>
+            <model_class>HistoryDatasetAssociation</model_class>
+            <test type="isinstance" test_attr="datatype" result_type="datatype">images.Jpg</test>
+            <test type="isinstance" test_attr="datatype" result_type="datatype">images.Png</test>
```

Comment:
@gregvonkuster do you know if this also works for tiff?

Reply:
I think read somewhere in the documentation that only jpg and png are currently supported simple image formats, but I cannot find it now.  Here is the page describing this  https://openseadragon.github.io/examples/tilesource-image/.  Perhaps a regular tiff image would work.  I don't remember if I tried loading one, but I'll do so.

I don't believe it works for pyramidal tiff images - https://github.com/openseadragon/openseadragon/issues/349, it seems that you have to convert them to dzi format - https://github.com/openseadragon/openseadragon/issues/351.  Animated gifs are also not supported https://ove.readthedocs.io/en/stable/ove-apps/packages/ove-app-images/README.html.